src/Kovan-DssSpell.sol

```diff
@@ -29,17 +37,131 @@ contract SpellAction {
     ChainlogAbstract constant CHANGELOG =
         ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
+    uint256 constant THOUSAND = 10**3;
+    uint256 constant MILLION = 10**6;
+    uint256 constant WAD = 10**18;
+    uint256 constant RAY = 10**27;
+    uint256 constant RAD = 10**45;
+
+    // DC IAM
     address constant MCD_DC_IAM = 0x0000000000000000000000000000000000000000;
 
+    // RENBTC-A
+    address constant RENBTC             = 0xe3dD56821f8C422849AF4816fE9B3c53c6a2F0Bd;
+    address constant MCD_JOIN_RENBTC_A  = 0x12F1F6c7E5fDF1B671CebFBDE974341847d0Caa4;
+    address constant MCD_FLIP_RENBTC_A  = 0x2a2E2436370e98505325111A6b98F63d158Fedc4;
+    address constant PIP_RENBTC         = 0x2f38a1bD385A9B395D01f2Cbf767b4527663edDB;
+    bytes32 constant RENBTC_ILK         = "RENBTC-A";
+
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    uint256 constant SIX_PERCENT_RATE = 1000000001847694957439350562;
+
     function execute() external {
         address MCD_VAT = CHANGELOG.getAddress("MCD_VAT");
+        address MCD_CAT = CHANGELOG.getAddress("MCD_CAT");
+        address MCD_JUG = CHANGELOG.getAddress("MCD_JUG");
+        address MCD_END = CHANGELOG.getAddress("MCD_END");
+        address MCD_SPOT = CHANGELOG.getAddress("MCD_SPOT");
+        address FLIPPER_MOM = CHANGELOG.getAddress("FLIPPER_MOM");
+        address ILK_REGISTRY = CHANGELOG.getAddress("ILK_REGISTRY");
+        address FAUCET = CHANGELOG.getAddress("FAUCET");
+        address OSM_MOM = CHANGELOG.getAddress("OSM_MOM");
 
         // Give permissions to the MCD_DC_IAM to file() the vat
         VatAbstract(MCD_VAT).rely(MCD_DC_IAM);
 
-        // Add MCD_DC_IAM to the changelog
+        // Add RENBTC-A ilk
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).ilk() == RENBTC_ILK, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).gem() == RENBTC, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).dec() == 8, "join-dec-not-match");
```

Comment:
Should this be `require(GemJoinAbstract(MCD_JOIN_RENBTC_A).dec() == ERC20(RENBTC).decimals())`?

I think we want to sanity check the join vs. the gem

Reply:
We can switch to that, but I think it was left out because there is no ERC20 interface with decimals defined.